src/libstore/globals.hh

```diff
@@ -122,6 +122,13 @@ public:
           for supported store types and settings.
         )"};
 
+    Setting<std::string> gcSocketPath {
+        this,
+        getEnv("NIX_GC_SOCKET_PATH").value_or("auto"),
```

Comment:
Now that we have `NIX_CONFIG`, are new `NIX_` env vars needed? I am not so sure.

Reply:
Fair point, it's not really needed indedd. I'll drop it